apps/voyager/assets/package.json

```diff
@@ -7,84 +7,66 @@
     "watch": "webpack --mode development --watch"
   },
   "dependencies": {
-    "@babel/runtime": "^7.12.5",
-    "@material-ui/core": "^4.10.2",
-    "@material-ui/icons": "^4.9.1",
-    "@material-ui/lab": "^4.0.0-alpha.56",
-    "@reduxjs/toolkit": "^1.4.0",
-    "axios": "^0.19.0",
-    "chart.js": "^2.9.3",
+    "@babel/runtime": "^7.12.13",
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.57",
+    "@reduxjs/toolkit": "^1.5.0",
+    "axios": "^0.21.1",
+    "chart.js": "^2.9.4",
     "chartjs-plugin-labels": "^1.1.0",
     "color-convert": "^2.0.1",
-    "diff": "^4.0.2",
-    "dompurify": "^2.1.0",
-    "file-saver": "^2.0.2",
-    "formik": "^2.1.5",
+    "diff": "^5.0.0",
+    "dompurify": "^2.2.6",
+    "formik": "^2.2.6",
     "google-palette": "^1.1.0",
-    "history": "^4.10.1",
-    "immutability-helper": "^3.0.1",
     "material-table": "^1.69.2",
-    "material-ui": "^0.20.2",
+    "moment": "^2.29.1",
```

Comment:
not necessarily worth the change but moment from my experience has a lot of issues. If you don't need timezone data, https://date-fns.org/ or https://moment.github.io/luxon/ may be better

Reply:
Okay, thanks for letting me know. Yeah I don't need timezone, so I should switch.